### ISSUE TITLE
About the re-implementation of Layernorm

### ISSUE THREAD

**Issue by ChongjianGE** [lucidrains/uniformer-pytorch#2]
Hi @lucidrains ,

Thanks for the implementation of Uniformer.
After checking the code, I found that you have re-implemented the layernorm operation in the following.
https://github.com/lucidrains/uniformer-pytorch/blob/78397000e647fd4035a7fa2bede999d59d4c3ded/uniformer_pytorch/uniformer_pytorch.py#L13-L23

I wonder if there are any differences between your re-implementation and the official code provided by PyTorch (i.e., torch.nn.Layernorm(dim)  ).

Looking forward to your reply. Thx.

**Comment by lucidrains** [comment]
@ChongjianGE no, there should be no difference, except that you get to save on a bunch of transposes

**Comment by ChongjianGE** [comment]
@lucidrains 
Get it. Thanks for the reply.
